css/style.css

```diff
@@ -701,8 +712,6 @@ label {
   }
 
   #line {
-    /* .shape1 { */
-
     width: 134px;
     height: 5px;
     background-color: #0d151e;
```

Comment:
- [ ] This wasn't the main topic of this project but it's worthy mentioning it as an optional requirement. The background and the whole site is distorted on the smallest screens (320px of width), though it was supposed to fit in it properly, I would recommend you to make sure that not only on this project but also in all of your future projects you should ensure that your pages work well in at least screens starting from 320px of width, kindly if you get enough time address this issue before you merge to the base branch. Check your page on 320px of width for more context. Thank you!
![Screenshot from 2021-12-14 09-33-59](https://user-images.githubusercontent.com/56886064/145953751-3600530c-2f5b-4d00-b9b7-756254817fdd.png)

Reply:
![oybek - mobile-ver-snapshot-footer](https://user-images.githubusercontent.com/85465559/145975687-43f787e9-3a81-4e35-9943-dfde1a6286f7.png)
We have fixed the issue as your feedback.
Thank you.